### ISSUE TITLE
CoRA Cat: Multiple boxes for mito haplogroup

### ISSUE THREAD

**Issue by btrnew** [spawaskar-cora/cora-docs#460]
Is it possible to make the Mito Haplo group section multiple boxes like the Match Count/Population Frequency section? When we receive information from AFDIL they don't always provide anything beyond the basic haplogroup (A, B, M, etc.) and scrolling through a giant list makes things a bit more difficult if all you need to input at this time is one letter. 

It would be helpful if it allowed you to input the basic haplogroup in the first box, the more refined haplogroup information in the second box, and population origin in the last box.

![CoRA Cat Mito haplogroup](https://user-images.githubusercontent.com/43450732/58676529-469f8000-82f4-11e9-816c-f2f337e14d31.PNG)
![CoRA Cat mito multiple boxes](https://user-images.githubusercontent.com/43450732/58676531-47d0ad00-82f4-11e9-8ee9-4fc704d53f29.PNG)

**Comment by clegarde** [comment]
Agreed. I thought we had discussed that the letter would be one and then the “subgroup” another?

**Comment by SachinPawaskarUNO** [comment]
Why? If one letter is all that is provided it should still be in the master list, which should be maintained by the project leads/managers. Eventually I would like to see this list come from AFDIL.
Also note that the list is sortable, so just type the first Letter.
Maybe we should discuss this in our SCRUM meetings. I'm not convinced as yet.

**Comment by clegarde** [comment]
If you’re not forced to choose a subgroup, then it should be fine.

**Comment by btrnew** [comment]
Some additional thoughts to consider for the CoRA scrum before I forget them: 

One reason it would be helpful to have separate entry boxes is for the purpose of report generation. With the current entry format, is it possible for me to generate an mtDNA report that requests all M haplogroups regardless of subgroups (for example)? .

Also, and this may be a DPAA business rules thing, as an analyst I am not allowed to assign population origin to specific genetic data provided to us by AFDIL. And, on the Tarawa project at least, we do not always receive population origin information when we receive DNA results. I could have all of the other information including the haplogroup and subgroup, but no assigned population origin per AFDIL. So in the current list format we would need duplicate entries for each haplogroup and subgroup that do not have "Asian", "European", etc. after them.

**Comment by btrnew** [comment]
Some additional thoughts to consider for the CoRA scrum before I forget them: 

One reason it would be helpful to have separate entry boxes is for the purpose of report generation. With the current entry format, is it possible for me to generate an mtDNA report that requests all M haplogroups regardless of subgroups (for example)? .

Also, and this may be a DPAA business rules thing, as an analyst I am not allowed to assign population origin to specific genetic data provided to us by AFDIL. And, on the Tarawa project at least, we do not always receive population origin information when we receive DNA results. I could have all of the other information including the haplogroup and subgroup, but no assigned population origin per AFDIL. So in the current list format we would need duplicate entries for each haplogroup and subgroup that do not have "Asian", "European", etc. after them.

**Comment by SachinPawaskarUNO** [comment]
@btrnew Can you please send me a list of all Haplogroup Letters you want to see being made available in CoRA?
@clegarde, To begin with, I could use the letters from the existing list that you have already created.
Let me know your thoughts?

**Comment by btrnew** [comment]
@SachinPawaskarUNO Here is a good link from one of the more widely used mtDNA research resources: https://www.phylotree.org/tree/index.htm

At this link you will see all of the various haplogroups we could possibly encounter and how they are related to one another. If you input all of the letters on the bottom row (ignore any astericks, just add the letter), as well as any of the letters under "incl." then you won't have anyone coming back later to say that the haplogroup they need isn't in CoRA unless some major change happens in the mtDNA world. 

The list that we already created only includes the haplogroups that we have seen in the various projects so far. While some of the haplogroups on that master list are unlikely to appear in the current DPAA projects, they could show up somewhere down the line as we expand into other regions of the world or any non-DPAA entities could request them. So, it might just be better to add them now and avoid annoyance on the part of the developers later.
